tiledb/libtiledb.pyx

```diff
@@ -3312,7 +3353,7 @@ cdef class Array(object):
             _raise_ctx_err(ctx_ptr, rc)
         return
 
-    def __init__(self, uri, mode='r', key=None, timestamp=None, Ctx ctx=default_ctx()):
+    def __init__(self, uri, mode='r', key=None, timestamp=None, attr=None, Ctx ctx=default_ctx()):
```

Comment:
do we want to generalize this to multiple attributes (one or more?)

Reply:
I could  add that and stub out the implementation for now... But we can provide the same thing semantically with queries. My reasoning for single-attribute views is to let a multi-attr object act like a regular numpy array.

Reply:
If we want to push the notion of arrays as views over some subset or all of the array it seems like it would generalize better.

Reply:
(discussed out-of-band: decision: go with current API for now and iterate in the future based on feedback)